app/models/account_warning.rb

```diff
@@ -10,14 +10,30 @@
 #  text              :text             default(""), not null
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
+#  report_id         :bigint(8)
+#  status_ids        :string           is an Array
 #
 
 class AccountWarning < ApplicationRecord
-  enum action: %i(none disable sensitive silence suspend), _suffix: :action
+  enum action: {
+    none:            0,
+    disable:         1_000,
+    delete_statuses: 1_500,
+    sensitive:       2_000,
+    silence:         3_000,
+    suspend:         4_000,
```

Comment:
No, they are updated in a migration. It is to prevent an event like when the force-sensitive option was added shifting existing records' enums by leaving enough space between values to insert new types.

Reply:
But inserting new types at the end isn't an issue, is it?

Reply:
Not using these weird values would mean we can skip the migration. Not a huge thing, but still.